crates/matrix-sdk-ui/src/timeline/tests/shields.rs

```diff
@@ -100,16 +100,14 @@ async fn test_local_sent_in_clear_shield() {
 
     // When the remote echo comes in.
     timeline
-        .handle_live_event(TimelineEvent::new(sync_timeline_event!({
-            "content": {
-                "body": "Local message",
-                "msgtype": "m.text",
-            },
-            "sender": &*ALICE,
-            "event_id": event_id,
-            "origin_server_ts": timestamp,
-            "type": "m.room.message",
-        })))
+        .handle_live_event(TimelineEvent::new(
+            EventFactory::new()
+                .text_msg("Local message")
+                .sender(*ALICE)
+                .event_id(event_id)
+                .server_ts(timestamp)
+                .into_raw_sync(),
+        ))
```

Comment:
The `EventBuilder` has a method to create a `TimelineEvent` out of it, so you can avoid a bit of boilerplate here. (Note that accepting this suggestion might require another `cargo fmt` to make the code look better.)

```suggestion
        .handle_live_event(
            EventFactory::new()
                .text_msg("Local message")
                .sender(*ALICE)
                .event_id(event_id)
                .server_ts(timestamp)
                .into_event(),
        )
```

Reply:
Nice, I've not yet delved into the different kinds of events produced by the different `into_*` methods on `EventBuilder`, so it was useful to learn a bit more about which ones to use when :slightly_smiling_face: Thanks for pointing this out, I'm happy to make this change!